compiler/rustc_middle/src/ty/layout.rs

```diff
@@ -933,14 +938,18 @@ impl<'tcx> LayoutCx<'tcx, TyCtxt<'tcx>> {
                             if let Bound::Included(start) = start {
                                 // FIXME(eddyb) this might be incorrect - it doesn't
                                 // account for wrap-around (end < start) ranges.
-                                assert!(*scalar.valid_range.start() <= start);
-                                scalar.valid_range = start..=*scalar.valid_range.end();
+                                assert!(scalar.valid_range.start <= start);
+                                // scalar.valid_range =
+                                //     AllocationRange { start, end: scalar.valid_range.end };
```

Comment:
The change you made seems good to me, so you can remove the commented out exact translation of the previous code